setup.py

```diff
@@ -32,6 +32,6 @@
     ],
     install_requires=[
         'ProxyTypes>=0.9',
-        'Django >=1.5, <1.7',
+        'Django >=1.5, <=1.8',
```

Comment:
I'd prefer <1.9 so it works for 1.8.1 without modifications, I'm comfortable there are unlikely to be breaking changes between minor versions.